src/spacel/provision/app/ingress_resource.py

```diff
@@ -102,3 +111,19 @@ def app_sg(self, orbit, region, app, sg_ref='Sg'):
                              orbit.name, region)
                 return None
             raise e
+
+    def app_eips(self, orbit, region, app, eips=[]):
+        cloudformation = self._clients.cloudformation(region)
+        stack_name = '%s-%s' % (orbit.name, app)
+        try:
+            index = len(eips) + 1
+            resource = cloudformation.describe_stack_resource(
```

Comment:
Calling `.describe_stack_resources()` once feels better than calling `.describe_stack_resource()` until it throws an error; 1 vs N server trips.

http://boto3.readthedocs.io/en/latest/reference/services/cloudformation.html#CloudFormation.Paginator.ListStackResources looks like the ultimate answer? no limits